src/elements/common/sub-header/ViewModeChangeButton.js

```diff
@@ -21,6 +22,7 @@ type Props = {
 
 const ViewModeChangeButton = ({ className = '', onViewModeChange, intl, viewMode, ...rest }: Props) => {
     const isGridView = viewMode === VIEW_MODE_GRID;
+    const iconColor = bdlNeutral03;
```

Comment:
Please avoid rename-only reassignments; we can pass the color value directly. Also update to use new color palette (bdlGray*)

Reply:
Made the change.  Want to ask though, is it ever a good idea to have rename-only reassignments? My thinking here was if there were more icons, and if someone wanted to change the color of all icons in the future, having `iconColor` would reduce the number of lines that would need to be changed.

Reply:
Renaming can almost always be accomplished during destructuring or at import. Changing the icon color in one place versus two seems like a minor optimization.

```
import { bdlGray01 as iconColor } from 'constants';
```